packages/playwright-core/src/server/browserType.ts

```diff
@@ -84,6 +85,37 @@ export abstract class BrowserType extends SdkObject {
     return browser._defaultContext!;
   }
 
+  async launchApp(options: {
+    sdkLanguage: string,
+    windowSize: types.Size,
+    windowPosition?: types.Point,
+    persistentContextOptions?: Parameters<BrowserType['launchPersistentContext']>[2];
+  }) {
+    const args = [...options.persistentContextOptions?.args ?? []];
+
+    if (this._name === 'chromium') {
```

Comment:
Do we upport non-chromium as recorder host?

Reply:
up we do.